K2Bridge.Tests.UnitTests/KustoDAL/ElasticResponseTests.cs

```diff
@@ -102,8 +102,10 @@ public string MapRowsToHits_WhenMultiWordDifferentCapsCapialLettersInQueryString
                 "somevalue1 somevalue2");
         }
 
+        // TODO: An empty elastic response should probably not contains any aggregations.
```

Comment:
can you add the issue link?

Reply:
https://dev.azure.com/CSECodeHub/476026%20-%20K2Bridge%20OSS%20-%20Visualization%20Capabilities/_workitems/edit/15049

Reply:
Link to the issue added to code.